includes/specials/SpecialBlockList.php

```diff
@@ -323,10 +328,11 @@ function formatValue( $name, $value ) {
 
 			case 'ipb_by':
 				if ( isset( $row->by_user_name ) ) {
-					$formatted = Linker::userLink( $value, $row->by_user_name );
-					$formatted .= Linker::userToolLinks( $value, $row->by_user_name );
+					$username = $this->usernameService->getUsername( $value, $row->by_user_name );
+					$formatted = Linker::userLink( $value, $username );
+					$formatted .= Linker::userToolLinks( $value, $username );
 				} else {
-					$formatted = htmlspecialchars( $row->ipb_by_text ); // foreign user?
+					$formatted = htmlspecialchars( $this->usernameService->getUsername( $row->ipb_id, $row->ipb_by_text ) ); // foreign user?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)